build-system/tasks/validator.js

```diff
@@ -33,6 +33,15 @@ async function validator() {
   });
 }
 
+/**
+ * Simple wrapper around the C++ validator tests
+ */
+async function validatorCppTest() {
+  execOrDie('bazel test validator_test', {
+    cwd: 'validator/cpp/engine',
+  });
+}
```

Comment:
This task function needs a description on line 65. If it makes sense to add an `--update_tests` flag similar to the main validator, you'd need to set `validatorCpp.flags` as well.

```js
validatorCpp.description = 'Builds and tests the AMP C++ validator.';
```

Reply:
Done.